### ISSUE TITLE
Content issue: sims in sec 22.11 physics is blank

### ISSUE THREAD

**Issue by Malar-Natarajan** [openstax/tutor#188]
**Describe the bug**
sims in sec 22.11 physics is blank

![Screen Shot 2018-12-17 at 1.07.00 PM.png](https://images.zenhubusercontent.com/5b23e2cc835fb47f522b81ba/53795c6c-340c-49ab-8f95-9b90c2b71c06)

**To Reproduce**

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**

**Comment by brittany-johnson** [comment]
[Link to card on devops bord](https://trello.com/c/FpuGTmtn/201-phet-sim-links-in-tutor-redirecting-to-un-secure-site)
cc: @carlycurran

**Comment by carlycurran** [comment]
Thanks, @brittany-johnson!

**Comment by scb6** [comment]
@nathanstitt confirmed it was working for him. Please move to Retest.

**Comment by nathanstitt** [comment]
Yes I now see it redirecting to the SSL url (https://archive.cnx.org/specials/5ca3e2cc-ae74-11e5-b6d3-f3c228f04b5c/magnet-and-compass/#sim-bar-magnet) as it should and then loads successfully

**Comment by Malar-Natarajan** [comment]
Verified in QA

**Comment by Malar-Natarajan** [comment]
Staging (and production)still doesn't have anything under the SIM
